main/src/main/java/org/mobilitydata/gtfsvalidator/notice/UnusedTripNotice.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Copyright should be MobilityData if Google didn't write the code.

Reply:
Indeed, thanks.